README.rst

```diff
@@ -425,9 +471,27 @@ Constructors for Sorted Containers
   the parameters of the type.  Ordering argument ``o`` is
   optional and defaults to ``Forward``.
 
+``SortedMultiDict(k,v,o)``
+  Construct a SortedMultiDict using keys given by ``k``, values
+  given by ``v`` and ordering object ``o``.  The ordering object
+  defaults to ``Forward`` if not specified.  The two arguments
+  ``k`` and ``v`` are 1-dimensional arrays of the same length in 
+  which ``k`` holds keys and ``v`` holds the corresponding values.
+  To construct an empty SortedMultiDict with given types ``K`` and
+  ``V``, use
+  ``SortedMultiDict(K[], V[], o)``.
```

Comment:
I believe this was in response to this comment: https://github.com/JuliaLang/DataStructures.jl/pull/88#issuecomment-106994292

See my response there.
(On github, it's attached inline to an unrelated comment).